platform/vs/asic_table.j2

```diff
@@ -0,0 +1 @@
+../mellanox/asic_table.j2
```

Comment:
Could you cut this dependency, and just use
```
    {
        "ASIC_TABLE:VS-ASIC": {
            "cell_size": "128",
            "pipeline_latency": "19",
            "mac_phy_delay": "0.8",
            "peer_response_time": "3.8"
        },
        "OP": "SET"
    }
```

Reply:
ok. will fix this.

Reply:
Sorry, I mean move the section out of `platform/mellanox/asic_table.j2` into `platform/vs/asic_table.j2`
Don't mix them in one file.

---
In reply to: [541482964](https://github.com/Azure/sonic-buildimage/pull/6194#discussion_r541482964) [](ancestors = 541482964)

Reply:
Ok. The vs part in Mellanox/asic table.j2 is no longer used and can be removed.
For the vs, we use asic table.json and asic table.j2 can be removed.